purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -166,55 +163,38 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         }, { errorLog("Error syncing purchases $it") })
     }
 
-    /**
-     * Add attribution data from a supported network
-     * @param [data] JSONObject containing the data to post to the attribution network
-     * @param [network] [AttributionNetwork] to post the data to
-     */
-    @Deprecated("use static addAttributionData", ReplaceWith("Purchases.addAttributionData(data, network)"))
-    fun addAttributionData(
-        data: JSONObject,
-        network: AttributionNetwork
-    ) {
-        postAttributionData(data, network, state.appUserID)
-    }
+    @JvmName("-deprecated_getEntitlements")
+    @Deprecated(
+        message = "moved to getOfferings()",
+        replaceWith = ReplaceWith(expression = "getOfferings(listener)"),
+        level = DeprecationLevel.ERROR)
+    fun getEntitlements() {
```

Comment:
Yes! We are deprecating and throwing a compilation error. I can't call getOfferings inside this method since the parameters differ. Also I had to remove the parameters of getEntitlements becuase the Entitlement class has been removed and the listener depended on it.

Reply:
👍